fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh

```diff
@@ -222,420 +233,433 @@ fi
 
 log_Msg "START"
 
-mkdir -p $WD
+#ScoutExtension must initialize for both cross-sectional and longitudinal modes.
 
-# Record the input options in a log file
-echo "$0 $@" >> $WD/log.txt
-echo "PWD = `pwd`" >> $WD/log.txt
-echo "date: `date`" >> $WD/log.txt
-echo " " >> $WD/log.txt
+case $DistortionCorrection in
+	${FIELDMAP_METHOD_OPT} | ${SIEMENS_METHOD_OPT} | ${GE_HEALTHCARE_LEGACY_METHOD_OPT} | ${GE_HEALTHCARE_METHOD_OPT} | ${PHILIPS_METHOD_OPT} | ${SPIN_ECHO_METHOD_OPT} ) ScoutExtension="_undistorted" 
+	;;
+	${NONE_METHOD_OPT}) ScoutExtension="_nosdc" 
+	;;
+esac
```

Comment:
```suggestion
case $DistortionCorrection in
	${FIELDMAP_METHOD_OPT} | ${SIEMENS_METHOD_OPT} | ${GE_HEALTHCARE_LEGACY_METHOD_OPT} | ${GE_HEALTHCARE_METHOD_OPT} | ${PHILIPS_METHOD_OPT} | ${SPIN_ECHO_METHOD_OPT} )
		ScoutExtension="_undistorted" 
	;;
	${NONE_METHOD_OPT})
		ScoutExtension="_nosdc" 
	;;
esac
```
Hard to see the "do this" part when it is on the end of a long line.